driver-ui/src/main/java/eu/tsystems/mms/tic/testframework/utils/FileDownloader.java

```diff
@@ -294,7 +291,8 @@ private String download(
             String cookieString,
             boolean useSecondConnection
     ) throws IOException {
-        LOGGER.info("Download " + url + " to " + targetFile.getAbsolutePath());
+
+        LOGGER.info("Start downloading " + url);
```

Comment:
Interface Loggable should prefered.

Reply:
This method is static and cannot use interface implementations.